.github/workflows/scripts/kubernetes/core/run-quickstart-test-on-kubernetes.sh

```diff
@@ -130,35 +104,15 @@ runPostHelmInstallCommands
 echo "running the tests"
 pwd
 
-route="localhost:8080"
-
-mvnVerifyArguments="-Dserver.host=${server_protocol}://${route} "
-extraMvnVerifyArguments="$(getMvnVerifyExtraArguments)"
-if [ -n "${extraMvnVerifyArguments}" ]; then
-  mvnVerifyArguments="${mvnVerifyArguments} ${extraMvnVerifyArguments}"
-fi
-if [ "${QS_DEBUG_TESTS}" = "1" ]; then
-  mvnVerifyArguments="${mvnVerifyArguments} -Dmaven.failsafe.debug=true"
-fi
-
-echo "Verify Arguments: ${mvnVerifyArguments}"
-
-mvn -B verify -Pintegration-testing ${mvnVerifyArguments}
-
-if [ "$?" != "0" ]; then
-  test_status=1
-  echo "Tests failed!"
-  echo "Dumping the application pod(s)"
-  kubectl logs deployment/"${application}"
-  testsFailed
-fi
+runningTests "${application}" "${server_protocol}" "$(getMvnVerifyExtraArguments)"
```

Comment:
I"m not sure I like 'hiding' this in the overridable-functions.

I think it would be cleaner to have some kind of check that can be overridden per tests in overridable-functions.sh, and an if/else check here. In the normal (if) case, run the tests normally in this file, and if we should run the tests ourselves implement that in the tests overridable-functions.sh.

BTW, I think a few other quickstarts use client/server, so if that can be a common use case which can be handled here, even better. Although perhaps that can come later.

Reply:
Thanks for your review, @kabir. I hope the last commit is close enough to having a default behaviour. Please, let me know what you think about it :-)